peps/pep-0757.rst

```diff
@@ -145,11 +145,18 @@ If :c:member:`PyLongExport.digits` is not ``NULL``, a private field of the
 
    Export a Python :class:`int` object.
 
-   On success, set *\*export_long* and return 0.
+   *export_long* must point to a :c:struct:`PyLongExport` structure allocated
+   by the caller.  It must not be ``NULL``.
+
+   On success, fill in *\*export_long* and return 0.
    On error, set an exception and return -1.
 
-   If *export_long->digits* is not ``NULL``, :c:func:`PyLong_FreeExport` must be
-   called when the export is no longer needed.
+   :c:func:`PyLong_FreeExport` must be called when the export is no longer
+   needed.
+
+    .. impl-detail::
```

Comment:
Can we do this with existing admonitions instead?

Reply:
Just write "Implementation detail: xxx".

Reply:
Victor's suggestion looks better, IMO.